ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java

```diff
@@ -98,6 +99,20 @@ public void setPagingEnabled(ReactHorizontalScrollView view, boolean pagingEnabl
     view.setPagingEnabled(pagingEnabled);
   }
 
+  /**
+   * Controls overScroll behaviour
+   */
+  @ReactProp(name = "overScrollMode")
+  public void setOverScrollMode(ReactHorizontalScrollView view, String value) {
+    if (value == null || value.equals(ReactScrollViewHelper.AUTO)) {
+      view.setOverScrollMode(View.OVER_SCROLL_IF_CONTENT_SCROLLS);
+    } else if (value.equals(ReactScrollViewHelper.OVER_SCROLL_ALWAYS)) {
+      view.setOverScrollMode(View.OVER_SCROLL_ALWAYS);
+    } else if (value.equals(ReactScrollViewHelper.OVER_SCROLL_NEVER)) {
+      view.setOverScrollMode(View.OVER_SCROLL_NEVER);
```

Comment:
Would it make sense to extract `convertOverscrollMode(String value)` (or similar name) to `ReactScrollViewHelper`? The three possible values are already defined there so the code would live close to them, and there would be no need to copy&paste these few lines in two places. I know it's just a bit of code but when someone's changing it (especially someone unfamiliar with the codebase) they might forget to do it in both places.